ledger/participant-integration-api/src/main/scala/platform/store/cache/MutableCacheBackedContractStore.scala

```diff
@@ -280,12 +281,12 @@ class MutableCacheBackedContractStore(
 object MutableCacheBackedContractStore {
   type EventSequentialId = Long
 
-  def apply(
+  private[cache] def apply(
       contractsReader: LedgerDaoContractsReader,
       signalNewLedgerHead: Offset => Unit,
       metrics: Metrics,
-      maxContractsCacheSize: Long = 100000L,
-      maxKeyCacheSize: Long = 100000L,
+      maxContractsCacheSize: Long = 100000,
```

Comment:
I have come to appreciate the way that the defaults are handled in the `ParticipantIndexerConfig` case with the source of truth close to the code in question

Reply:
Removed the defaults from here and bubbled them up in the CLI config.